src/main/java/teammates/ui/webapi/CreateFeedbackSessionAction.java

```diff
@@ -74,4 +82,37 @@ public JsonResult execute() {
         return new JsonResult(output);
     }
 
+    private void createFeedbackQuestions(String copyCourseId, String newCourseId, String feedbackSessionName,
+            String oldSessionName) {
+        logic.getFeedbackQuestionsForSession(oldSessionName, copyCourseId).forEach(question -> {
+            FeedbackQuestionAttributes attributes = FeedbackQuestionAttributes.builder()
+                    .withCourseId(newCourseId)
+                    .withFeedbackSessionName(feedbackSessionName)
+                    .withGiverType(question.getGiverType())
+                    .withRecipientType(question.getRecipientType())
+                    .withQuestionNumber(question.getQuestionNumber())
+                    .withNumberOfEntitiesToGiveFeedbackTo(question.getNumberOfEntitiesToGiveFeedbackTo())
+                    .withShowResponsesTo(question.getShowResponsesTo())
+                    .withShowGiverNameTo(question.getShowGiverNameTo())
+                    .withShowRecipientNameTo(question.getShowRecipientNameTo())
+                    .withQuestionDetails(question.getQuestionDetails())
+                    .withQuestionDescription(question.getQuestionDescription())
+                    .build();
+
+            FeedbackQuestionDetails questionDetails = attributes.getQuestionDetails();
+            if (questionDetails instanceof FeedbackMsqQuestionDetails) {
+                FeedbackMsqQuestionDetails msqQuestionDetails = (FeedbackMsqQuestionDetails) questionDetails;
+                int numOfGeneratedMsqChoices = logic.getNumOfGeneratedChoicesForParticipantType(
+                        attributes.getCourseId(), msqQuestionDetails.getGenerateOptionsFor());
+                msqQuestionDetails.setNumOfGeneratedMsqChoices(numOfGeneratedMsqChoices);
+                questionDetails = msqQuestionDetails;
+            }
+
+            try {
+                attributes = logic.createFeedbackQuestion(attributes);
+            } catch (InvalidParametersException e) {
+                throw new InvalidHttpRequestBodyException(e.getMessage(), e);
```

Comment:
A valid question in one session should be valid in any other session, so if this happens, something is not right and it's not due to user. Need a `log.severe` here instead of throwing this exception.